### PR TITLE
Update on conclave mods

### PR DESCRIPTION
**Pull request by lonnstyle**
some optimizations are done to extra items
`Arquebex` is deleted from `rivens` folder as it is an exalted weapon

### CODE REVIEW

File: tracked/items/Follow Through.json [hunk 1]
```diff
@@ -7,18 +7,18 @@
         "uncommon",
         "warframe"
     ],
-    "icon": "icons/en/Follow_Through.c4f54158e0f340645e9f79d974dba7d9.png",
+    "icon": "https://static.wikia.nocookie.net/warframe/images/b/b8/FollowThroughMod.png",
     "thumb": "icons/en/thumbs/Follow_Through.c4f54158e0f340645e9f79d974dba7d9.128x128.png",
     "icon_format": "port",
-    "url_name": "follow_through",
+    "url_name": "Follow_Through",
```
Comment (KycKyc) [inline tracked/items/Follow Through.json]: do not change `url_name`, otherwise it will change url of the page

**Review by KycKyc — CHANGES_REQUESTED** [review]
All good, except `Flow Through`

File: tracked/items/Follow Through.json (hunk 1 above)
Comment (lonnstyle) [reply tracked/items/Follow Through.json]: Fixed!

**Review by KycKyc — APPROVED** [review]
Thanks !

**Pull request merged**